File: web/src/main/java/by/sorface/passport/web/converters/PrincipalConverter.java
package by.sorface.passport.web.converters;

import by.sorface.passport.web.dao.models.UserEntity;
import by.sorface.passport.web.records.principals.DefaultPrincipal;
import org.springframework.core.convert.converter.Converter;

public interface PrincipalConverter extends Converter<UserEntity, DefaultPrincipal> {
}

File: web/src/main/kotlin/by/sorface/passport/web/security/converters/PrincipalConverter.kt
package by.sorface.passport.web.security.converters

import by.sorface.passport.web.dao.sql.models.UserEntity
import by.sorface.passport.web.records.principals.DefaultPrincipal
import org.springframework.core.convert.converter.Converter

interface PrincipalConverter : Converter<UserEntity, DefaultPrincipal>
